### PR TITLE
Create basic components

### PR DESCRIPTION
**Pull request by apomelitos**

### CODE REVIEW

File: src/components/Grid.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  cols: number;
+};
+
+export const Grid: FC<GridProps> = ({ rows, cols }): JSX.Element => {
+  return (
+    <div className='grid'>
+      {Array(rows * cols)
+        .fill(0)
+        .map((elem, idx) => {
+          return <div key={`${idx}`} className={`grid__cell`} />;
```
Comment (mrlika) [inline src/components/Grid.tsx]: Simplify to: `<div key={idx} className='grid__cell' />;`

File: src/components/Game.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { FC } from 'react';
+import { Board } from './Board';
+
+const ROWS = 4,
+  COLS = 4;
```
Comment (mrlika) [inline src/components/Game.tsx]: We don't merge variables declarations. Use `const` twice.

**Review by mrlika — CHANGES_REQUESTED** [review]

File: src/components/Board.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  rows: number;
+};
+
+export const Board: FC<BoardProps> = ({ cols, rows }): JSX.Element => {
+  const tileTotalWidth = 100;
+  const boardPadding = 10;
+
+  const containerWidht = tileTotalWidth * cols;
+
+  const boardWidth = containerWidht + boardPadding * 2;
```
Comment (TheForsakenSpirit) [inline src/components/Board.tsx:15]: All this constant don't changes between renders. Move from function.

File: src/components/Game.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import { Board } from './Board';
+
+const ROWS = 4;
+const COLS = 4;
+
+export const Game: FC = (): JSX.Element => {
+  return (
+    <div>
+      <Board cols={COLS} rows={ROWS} />
+    </div>
```
Comment (TheForsakenSpirit) [inline src/components/Game.tsx:11]: Macke two component:
- Background (or real board).
- Cells(or something like this) that will represent cells.

File: src/components/Grid.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  cols: number;
+};
+
+export const Grid: FC<GridProps> = ({ rows, cols }): JSX.Element => {
+  return (
+    <div className='grid'>
+      {Array(rows * cols)
+        .fill(0)
+        .map((elem, idx) => (
+          <div key={idx} className='grid__cell' />
```
Comment (TheForsakenSpirit) [inline src/components/Grid.tsx:15]:
```suggestion
          <div key={idx} className='grid-cell' />
```
For class names we use cebab-case.

**Review by TheForsakenSpirit — CHANGES_REQUESTED** [review]

File: src/components/Game.tsx (hunk 4 above)
Comment (apomelitos) [reply src/components/Game.tsx:11]: Should i make it inside Board component or instead of Board component?

**Pull request merged**